sources/web/datalab/static/websocket.js

```diff
@@ -1,6 +1,9 @@
 define(['util'], (util) => {
 
   function shouldShimWebsockets() {
+    if (appSettings.shimWebsockets) {
```

Comment:
This variable isn't available in the UI.

I would change this to something like `if (document.body.getAttribute('data-proxy-web-sockets') == 'true') {`, and then add `data-proxy-web-sockets` to the HTML template and fill it in based on the `appSettings.shimWebsockets` setting.